dual_net.py

```diff
@@ -301,61 +312,39 @@ def logging_ops():
         name="weight_summaries")
 
 
+def compute_gradient_ratio(weight_tensors, before_weights, after_weights):
+    """Compute the ratio of gradient norm to weight norm."""
+    deltas = [after - before for after, before in zip(after_weights, before_weights)]
+    delta_norms = [np.linalg.norm(d.ravel()) for d in deltas]
+    weight_norms = [np.linalg.norm(w.ravel()) for w in before_weights]
+    ratios = [d / w for d, w in zip(delta_norms, weight_norms)]
+    all_summaries = [tf.Summary.Value(tag=tensor.name, simple_value=ratio)
```

Comment:
So the tensor name is the summary tag? Maybe append something like "update_ratio" to be more clear?